Usiqee/Model/Author/Author.swift

```diff
@@ -8,13 +8,99 @@
 import FirebaseFirestoreSwift
 
 struct Author: Identifiable, Codable {
+    
+    // MARK: - Enum
+    enum SocialMedia: Int {
+        case twitter = 1
+        case facebook
+        case instagram
+        case youtube
+        case snapchat
+        case twitch
+        case tiktok
+        case linkedin
+        
+        var image: UIImage? {
+            switch self {
+            case .twitter:
+                return UIImage(named: "twitter")
+            case .facebook:
+                return UIImage(named: "facebook")
+            case .instagram:
+                return UIImage(named: "instagram")
+            case .youtube:
+                return UIImage(named: "youtube")
+            case .snapchat:
+                return UIImage(named: "snapchat")
+            case .twitch:
+                return UIImage(named: "twitch")
+            case .tiktok:
+                return UIImage(named: "tiktok")
+            case .linkedin:
+                return UIImage(named: "linkedin")
+            }
+        }
+    }
```

Comment:
should be rename `AuthorSocialMedia` or put in a new file `SocialMedia`

Reply:
it already inside author (should be prefixed by Author `Author.SocialMedia`)